<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvaluateForm</class>
 <widget class="QWidget" name="EvaluateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,9,0" columnstretch="0,5,3,0,0">
   <item row="1" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="PlotForm" name="ePlotWidget_01" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="PlotForm" name="ePlotWidget_02" native="true"/>
     </item>
     <item row="0" column="2">
      <widget class="PlotForm" name="ePlotWidget_03" native="true"/>
     </item>
     <item row="0" column="3">
      <widget class="PlotForm" name="ePlotWidget_04" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="PlotForm" name="ePlotWidget_05" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="PlotForm" name="ePlotWidget_06" native="true"/>
     </item>
     <item row="1" column="2">
      <widget class="PlotForm" name="ePlotWidget_07" native="true"/>
     </item>
     <item row="1" column="3">
      <widget class="PlotForm" name="ePlotWidget_08" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="PlotForm" name="ePlotWidget_09" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="PlotForm" name="ePlotWidget_10" native="true"/>
     </item>
     <item row="2" column="2">
      <widget class="PlotForm" name="ePlotWidget_11" native="true"/>
     </item>
     <item row="2" column="3">
      <widget class="PlotForm" name="ePlotWidget_12" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="PlotForm" name="ePlotWidget_13" native="true"/>
     </item>
     <item row="3" column="1">
      <widget class="PlotForm" name="ePlotWidget_14" native="true"/>
     </item>
     <item row="3" column="2">
      <widget class="PlotForm" name="ePlotWidget_15" native="true"/>
     </item>
     <item row="3" column="3">
      <widget class="PlotForm" name="ePlotWidget_16" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="Icons/Icons.qrc">
         <normaloff>:/application_add.png</normaloff>:/application_add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Analyzer State 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Analyzer State 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Analyzer State 3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Analyzer State 4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="3" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="MatrixForm" name="matrixWidget" native="true"/>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QToolButton" name="magnifyToolButton">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icons/Icons.qrc">
         <normaloff>:/magnifier_zoom_in.png</normaloff>:/magnifier_zoom_in.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Polarizer State 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Polarizer State 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Polarizer State 3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Polarizer State 4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotForm</class>
   <extends>QWidget</extends>
   <header location="global">plotform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MatrixForm</class>
   <extends>QWidget</extends>
   <header location="global">matrixform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
